test/cmake projects/test proj/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ add_subdirectory(app)
 # Disabled by default library which has some compilations errors
 add_subdirectory(incorrect_lib)
 
+add_executable(MakefileDemo makefile_main.cpp)
```

Comment:
Makefile is weird in this way since `app/main.cpp` isn't a top level target, app/ is a subdirectory so root-level invocation of `make` doesn't see it so I had to create a new executable just for testing.